codice/ddf
# [2.26.x] Added new session property for Java subject to fix Hawtio

#### What does this PR do?

#### Who is reviewing it? 
@ryeats
@smithjosh

#### Select relevant component teams: 
@codice/security 

#### Ask 2 committers to review/merge the PR and tag them here.
@ryeats
@smithjosh

#### How should this be tested?
Install Hawtio
browse to https://localhost:8993/hawtio
see that default DDF login pops up
able to see/use Hawtio page after successful login

#### Any background context you want to provide?

#### What are the relevant tickets?
Fixes: #6581 

#### Screenshots


#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
